### ISSUE TITLE
New page: All goals overview

### ISSUE THREAD

**Issue by tsteur** [piwik/piwik#9625]
We already have a page called "All websites" overview.  I'm suggesting to create a new page "All goals". 

As explained [here](https://github.com/piwik/piwik/issues/9624#issue-128349250) goal conversions are very important when someone is using them. Possibly more than any other metrics. As such it would be valuable to be able to see an overview of all goals of all your websites (or a selected bunch of websites). 

I know this can be slow to load and not everyone is using goals. Therefore it is not usable for all Piwik installations so it might be good to put it as a plugin on the Marketplace. 

Basically the page could be similar to this:

![image](https://cloud.githubusercontent.com/assets/273120/12532518/ef4d8de8-c27a-11e5-8dbf-de507e44b714.png)

When clicking on a metric next to a sparkline it would be displayed on the big graph instead of the tiny sparkline so one can actually compare goal conversions.

**Comment by tsteur** [comment]
Similar to https://github.com/piwik/piwik/issues/7543#issuecomment-174221949 it might be good to have different visualizations for this screen. A plain table might be good, only a graph and a graph + table.
